docs/contents/user-docs/authorization.md

```diff
@@ -0,0 +1,145 @@
+# Authorization
+
+When we talk about authorization and access control for ACK, we need to discuss
+two different RBAC systems.
+
+Remember that [Kubernetes RBAC][0] governs a **Kubernetes** user's ability to
+read or write **Kubernetes resources**. In the case of ACK, this means that
+Kubernetes RBAC system controls the ability of a Kubernetes user to read or
+write different custom resources (CRs) exposed by ACK service controllers.
+
+[0]: https://kubernetes.io/docs/reference/access-authn-authz/authorization/
+
+On the other end of the authorization spectrum, AWS IAM has an RBAC system
```

Comment:
This is confusing to me because IAM is more than RBAC. Maybe users don't care but it may be confusing for environments that use IAM for attribute based access control (ABAC).

Reply:
good point. I've clarified.